.github/workflows/upstream.yml

```diff
@@ -23,3 +24,19 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm run upstream
+      - name: Handle error
+        if: ${{ failure() }}
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            const { owner, repo } = context.repo;
+            const nodeVersion = process.version;
+            const issueTitle = `Upstream check failed`;
+            const issueBody = `Node ${nodeVersion} has made changes to the functions that depend on this module. Please review the new function signatures and make the corresponding updates to this module.`;
```

Comment:
```suggestion
            const issueBody = `Node ${nodeVersion} has made changes to the functions this module depends on. Please review the new function changes, determine if this module requires changes because of them, and update the function hashes. Add a comment below with your findings.`;
```